test/DbConnectedResourceAbstractFactoryTest.php

```diff
@@ -6,49 +6,59 @@
 
 namespace ZFTest\Apigility;
 
+use Interop\Container\ContainerInterface;
 use PHPUnit_Framework_TestCase as TestCase;
 use stdClass;
```

Comment:
Nevermind, I thought you were pointing out the `ContainerInterface`, not `stdClass`. I'll update. (It was used before I refactored the tests.)